internal/broker/services_test.go

```diff
@@ -115,6 +115,166 @@ func TestServices_Services(t *testing.T) {
 		assertPlansContainPropertyInSchemas(t, services[0], "oidc")
 		assertPlansContainPropertyInSchemas(t, services[0], "administrators")
 	})
+
+	t.Run("should containt the property 'required' with values [name region] when ExposeSchemaWithRegionRequired is true and RegionParameterIsRequired is false", func(t *testing.T) {
```

Comment:
Any unit tests added in this PR may be removed after this feature goes live on all environments.